packages/cli/src/cmds/preview/commonUtils.ts

```diff
@@ -340,7 +340,16 @@ export function mergeProcessEnv(
   }
   const result = Object.assign({}, process.env);
   for (const key of Object.keys(env)) {
-    result[key] = env[key];
+    if (key === "PATH") {
+      const pathKey = Object.keys(result).find((value) => value.toUpperCase() === "PATH");
+      if (pathKey !== undefined) {
+        result[pathKey] = `${env[key]}${path.delimiter}${result[pathKey]}`;
```

Comment:
It's wired to append here (this func `mergeProcessEnv` is to add or override envs)... just override, and let caller to input the full value.

Or, just apply the same logic as vscode.

Reply:
You are right, it is wired..
Updated to use the same logic as vscode: https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/terminal/common/terminalEnvironment.ts#L19.